util/kube/util.go

```diff
@@ -70,8 +72,13 @@ func (ku *kubeUtil) CreateOrUpdateSecret(ns string, name string, update updateFn
 }
 
 // CreateOrUpdateSecretField creates or updates a secret name in namespace ns, with given value for given field
+// when field value is null.
 func (ku *kubeUtil) CreateOrUpdateSecretField(ns string, name string, field string, value string) error {
 	err := ku.CreateOrUpdateSecret(ns, name, func(s *apiv1.Secret, new bool) error {
+		data, exist := s.Data[field]
```

Comment:
It's not ideal to change a public method like this if any other projects happen to be importing it.

Reply:
I checked the code,the func is only used by init.Howerver, this might be a potential risk if later developers don't pay attention.